### PR TITLE
Toolbar and headline.

### PR DESCRIPTION
**Pull request by 0sugo**
In this PR I did the following;
Created a navigation bar using flexbox.
Added a background image.
Added  text on the landing page about myself
Added social media icons and positioned them using flexbox.
Attached a well-detailed readme file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <title>Osugo</title>
+</head>
+
+<body>
+    <header>
+        <nav class="menu">
+            <ul>
+                <li id="logo">Joseck</li>
+                <li id="menu-icon"><i class="fa-solid fa-bars"></i></li>
+            </ul>
```
Comment (emmiiorji) [inline index.html]: Going by semantics, the logo and the hamburger menu do not really make a list so I'll recommend that you remove them from with `ul` and `li` elements. Instead, kindly let the logo be in an anchor (`a`)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    </header>
+    <div class="start">
+        <main>
+
+            <section class="banner">
+                <h1>Hey there. I'm Osugo. <br> <span>I'm a Software Developer</span></h1>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If
+                    you
+                    like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </section>
```
Comment (emmiiorji) [inline index.html]: You seem not to be using the exact same font as the figma design for the headline section. Kindly use the same font ('Inter') as used on the figma design.

Your design:
image

Figma design:
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-medium"></i></a></li>
+            </ul>
+        </div>
+
+        <div id="chev">
+            <i class="fa-solid fa-circle-chevron-down"></i>
+        </div>
```
Comment (emmiiorji) [inline index.html]: Kindly ensure that the social media icons maintain similar margin as given on the design.

Also. the little arrow below the social icons seems not to match that given in the design. Could you modify it to match as closely as possible in order to follow the design closely?

Your design:
image

Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @0sugo, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Proper use of GitHub flow


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @0sugo , I'm @MaganaAsiati , your reviewer for this PR  :wave: 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Descriptive pull request :heavy_check_mark:👨🏻‍💻
- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- [ ] Check the comments under the review.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Cheers and Happy coding! 🤓 👏👏👏

**Pull request merged**